manifest.json

```diff
@@ -0,0 +1,360 @@
+{
+  "id" : "org.libvips.vipsdisp",
```

Comment:
and the manifest file should be named afer the app-id. I didn't notice that's why the build failed.

Reply:
Ah I copied `build-dir/files/manifest.json` for this PR, I was confused.

I'll copy my `org.libvips.vipsdisp.json` instead.